lib/ace/mode/php_completions.js

```diff
@@ -9374,11 +9374,10 @@ var PhpCompletions = function() {
 
     this.getArrayKeyCompletions = function(state, session, pos, prefix) {
         var line = session.getLine(pos.row).substr(0, pos.column);
-        /(\$[\w]*)\[["']([^'"]*)$/i.test(line);
-        var variable = RegExp.$1;
+        var variable = line.match(/(\$[\w]*)\[["']([^'"]*)$/i)[1];
 
```

Comment:
@nightwing It has been checked in line 9344, so... I think the match has no chance to be null, is that right?